tests/common/dualtor/dual_tor_io.py

```diff
@@ -223,17 +232,13 @@ def generate_from_t1_to_server(self):
                     eth_src = self.ptfadapter.dataplane.get_mac(
                         0, ptf_t1_src_intf
                     )
-                tcp_tx_packet = testutils.simple_tcp_packet(
-                    eth_dst=eth_dst,
-                    eth_src=eth_src,
-                    ip_dst=server_ip,
-                    ip_src=self.random_host_ip(),
-                    ip_ttl=ip_ttl,
-                    tcp_dport=TCP_DST_PORT
-                )
-                payload = str(i) + 'X' * 60
-                packet = scapyall.Ether(str(tcp_tx_packet))
+                    packet[scapyall.Ether].src = eth_src
+                packet[scapyall.IP].src = self.random_host_ip()
+                packet[scapyall.IP].dst = server_ip
+                payload = str(i) + payload_suffix
                 packet.load = payload
+                packet[scapyall.IP].chksum = None
```

Comment:
Is this added because we are going to verify packet checksums? Why is this None?
Also, can this be moved  above where `tcp_tx_packet_orig ` is defined.

Reply:
I believe scapy calculates the checksum at packet creation. When we make a copy of the packet and change the attributes, the checksum will be incorrect so setting it to None will erase it.